buidl/mnemonic.py

```diff
@@ -98,6 +99,40 @@ def bytes_to_mnemonic(b, num_bits):
     return " ".join(mnemonic)
 
 
+def dice_rolls_to_mnemonic(dice_rolls, num_words=24, allow_low_entropy=False):
+    """
+    returns a mnemonic from a string of 6-sided dice rolls
+    (>=100 rolls for 24 words recommended)
+    (>=50 rolls for 12 words recommended)
+    """
+    # check that the number of words provided is 12, 15, 18, 21, or 24
+    if num_words not in (12, 15, 18, 21, 24):
+        raise InvalidBIP39Length(
+            f"{num_words} words requested (must be 12, 15, 18, 21, or 24 words)"
+        )
+    # check that valid dice rolls have been provided
+    if not isinstance(dice_rolls, str):
+        raise ValueError("Dice rolls must be provided as a string")
+    if (
+        len([roll for roll in dice_rolls if roll not in ("1", "2", "3", "4", "5", "6")])
+        > 0
+    ):
+        raise ValueError("Dice roll string contained invalid dice numbers")
+    # entropy (in bits) per 6-sided dice roll (i.e. log2(6)=2.585)
+    entropy_per_roll = 2.585
+    mnemonic_entropy = {12: 128, 15: 160, 18: 192, 21: 224, 24: 256}
+    # check that the appropriate amount of entropy had been provided
+    min_rolls_needed = ceil(mnemonic_entropy[num_words] / entropy_per_roll)
```

Comment:
While technically correct (and great to see the code on how it's done), this would be easier to read for developers who are not cryptographically inclined if you included the calculation results in the `mnemonic_entropy` dict above. Otherwise, if you get an error for an where `num_words` is `12` you don't know how many rolls are needed for another entry where `num_words` is 24.

Something like
```
entropy_calcs_dict = {
    # num_words: [bits_of_entropy, min_rolls_needed],
    12: [128, 50],
    15: [160, 62],
    18: [192, 75],
    21: [224, 87],
    24: [256, 100],
}
# min_rolls_needed is calculated as ceil(num_words / entropy_per_roll), where entropy_per_roll is log2(6)=2.585
```